cypress/integration/Tab/Is_clickable/index.js

```diff
@@ -19,3 +29,9 @@ Then('the onClick handler is called', () => {
         expect(win.onClick).to.be.calledWith({})
     })
 })
+
+Then('the onClick handler is not called', () => {
+    cy.window().then(win => {
+        expect(win.onClick).not.to.be.called
+    })
+})
```

Comment:
Perhaps add something to assert the `selected` prop. The selected tab needs to have a big blue underline.

Reply:
Agreed, though also here I'd prefer visual tests.

Reply:
This is again style related, so not part of the PR scope. I'll resolve this one.